modules/gdscript/gdscript_vm.cpp

```diff
@@ -579,9 +579,8 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 
 #ifdef DEBUG_ENABLED
 
-	if (EngineDebugger::is_active()) {
-		GDScriptLanguage::get_singleton()->enter_function(p_instance, this, stack, &ip, &line);
-	}
+	GDScriptLanguage::CallLevel call_level;
+	GDScriptLanguage::get_singleton()->enter_function(&call_level, p_instance, this, stack, &ip, &line);
```

Comment:
This whole thing with keeping a pointer to this `call_level` here seems to cause some trouble with coroutines (functions that `await`), as they're allowed to exit this method (causing this pointer to go stale) without calling `GDScriptLanguage::exit_function` first, since that gets called as part of `GDScriptFunctionState::resume` instead.